src/content/tutorials/access-management/iam-roles-for-service-accounts/index.md

```diff
@@ -0,0 +1,123 @@
+
+The official documentation from AWS: [IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) `IRSA`.
+
+## IAM role permissions on `capa` controller
+
+The Cluster APi for AWs controller comes with an attached policy name `giantswarm-<INSTALLATION>-irsa-controller-policy` that contains permissions like:
+
+```json
+  {
+      "Effect": "Allow",
+      "Action": [
+        "iam:CreateOpenIDConnectProvider",
+        ...
+        "cloudfront:TagResource",
+        ...
+        "acm:RequestCertificate",
+        ...
+      ],
+      "Resource": "*"
+  },
+  {
+      "Effect": "Allow",
+      "Action": [
+        "s3:CreateBucket",
+        ...
+      ],
+      "Resource": "arn:aws:s3:::*-g8s-*"
+  }
+```
+
+This policy allows the controller to create a Cloudfront distribution, ACM certificate, S3 bucket and OpenID Connect provider to enable the usage of the IRSA feature.
+
+## Using IAM roles for service accounts
+
+### IAM role
+
+#### IAM Role Trusted Entity
+
+The **trusted entity** of a IAM role needs to contain a condition like the `sub` service account name of a JSON Web token (JWT). You need to ensure one of those conditions match against the `JWT`, which is used by one of your applications.
+
+This is a example `JWT` from `external-dns` which is verified against the `external-dns` IAM role.
+
+```json
+{
+  "aud": [
+    "sts.amazonaws.com"
+  ],
+  "kubernetes.io": {
+    "namespace": "kube-system",
+    "pod": {
+      "name": "external-dns-7dfdc97c55-v5spg",
+    },
+    "serviceaccount": {
+      "name": "external-dns",
+    }
+  },
+  "sub": "system:serviceaccount:kube-system:external-dns"
+}
+```
+
+### Service account
+
+The service account has to be annotated with a full `ARN` of the IAM role. You can get the `ARN` of the role in the AWS console, when you check role details.
+
+The annotation key has to be `eks.amazonaws.com/role-arn`.
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::$AWS_ACCOUNT:role/$IAM_ROLE_NAME
+  name: s3-access
+  namespace: default
+```
+
+## Cross Account Roles {#cross-account}
+
+### IAM role cross account
+
+In order to assume roles cross AWS accounts you will need to create a new AWS Identity Provider (OpenID Connect) in the AWS account where the IAM role is located.
+
+![Creating AWS Identity Provider](identity-provider.png)
+
+Login into the AWS on the account where the cluster is running:
+
+- Grab the address of the `Identity Provider` in your current cluster `IAM > Identity Providers`. It will look like `https://s3.REGION.amazonaws.com/AWS_ACCOUNT-g8s-CLUSTER_ID-oidc-pod-identity-v2` or `https://CLOUDFRONT_ID.cloudfront.net`.
```

Comment:
```suggestion
- Grab the address of the `Identity Provider` in your current cluster via [AWS Console > IAM > Identity Providers](https://us-east-1.console.aws.amazon.com/iam/home?#/identity_providers). It will look like `https://irsa.mycluster.<basedomain>`.
```

(for China, we can adapt this later)